sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -176,21 +176,86 @@ ALTER TABLE `track_events` DROP INDEX `unique_event_label_url`;
-  `updated_by` BIGINT(20) COMMENT 'fk to users.id for user who last updated this record',
-  PRIMARY KEY (`id`),
-  UNIQUE KEY `uuid` (`uuid`)
-) ENGINE=InnoDB;
+    `id`           int(11) NOT NULL AUTO_INCREMENT,
+    `uuid`         binary(16)   DEFAULT NULL,
+    `pid`          int(11) NOT NULL COMMENT 'fk to patient_data.pid',
+    `status`       varchar(100) DEFAULT 'active' COMMENT 'fk to list_options.option_id where list_id=Care_Team_Status',
+    `team_name`    varchar(255) DEFAULT NULL,
+    `note`         text,
+    `date_created` datetime     DEFAULT current_timestamp(),
+    `date_updated` datetime     DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+    `created_by`   BIGINT(20) COMMENT 'fk to users.id for user who created this record',
+    `updated_by`   BIGINT(20) COMMENT 'fk to users.id for user who last updated this record',
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `uuid` (`uuid`)
+) ENGINE = InnoDB;
+
+CREATE TABLE `care_team_member` (
+    `id`             int(11)     NOT NULL AUTO_INCREMENT,
+    `care_team_id`   int(11)     NOT NULL,
+    `user_id`        BIGINT(20) COMMENT 'fk to users.id represents a provider or staff member',
+    `contact_id`     BIGINT(20) COMMENT 'fk to contact.id which represents a contact person not in users or facility table',
+    `role`           varchar(50) NOT NULL COMMENT 'fk to list_options.option_id WHERE list_id=care_team_roles',
+    `facility_id`    BIGINT(20) COMMENT 'fk to facility.id represents an organization or location',
+    `provider_since` date        NULL,
+    `status`         varchar(100) DEFAULT 'active' COMMENT 'fk to list_options.option_id where list_id=Care_Team_Status',
+    `date_created`   datetime     DEFAULT current_timestamp(),
+    `date_updated`   datetime     DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+    `created_by`     BIGINT(20) COMMENT 'fk to users.id and is the user that added this team member',
+    `updated_by`     BIGINT(20) COMMENT 'fk to users.id and is the user that last updated this team member',
+    `note`           text,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `care_team_member_unique` (`care_team_id`, `user_id`, `facility_id`, `contact_id`)
+) ENGINE = InnoDB COMMENT ='Stores members of a care team for a patient';
+
+-- Migrate existing care teams. Thank you, Claude.ai for the help.
+INSERT INTO `care_teams` (`uuid`, `pid`, `status`, `team_name`, `date_created`, `date_updated`)
+SELECT
+    `uuid`,
+    `pid`,
+    `status`,
+    `team_name`,
+    MIN(`date_created`) AS `date_created`,
+    MAX(`date_updated`) AS `date_updated`
+FROM `care_teams_old`
+GROUP BY `pid`, `team_name`, `status`;
+
+-- Step 5: Migrate all members to care_team_member table
+INSERT INTO `care_team_member` (
+    `care_team_id`,
+    `user_id`,
+    `role`,
+    `facility_id`,
+    `provider_since`,
+    `status`,
+    `date_created`,
+    `date_updated`,
+    `note`
+)
+SELECT
+    ct_new.`id` AS `care_team_id`,
```

Comment:
I'm a bit confused, isn't the care_teams we want to migrate from the patient_data.care_team_provider, patient_data.care_team_facility and patient_data.care_team_status?

We would also migrate patient_history.care_team_provider and patient_history.care_team_facility?

I have an idea for how I'd do this so I'll put something together, I don't think we can do it with a standard SQL query due to the denormalized pipe (|) syntax we use for this.  So I'll put together a SQLUpgrade function to handle it.